packages/next/index.js

```diff
@@ -271,6 +279,19 @@ export class NextStackable extends BaseStackable {
 
     return scripts
   }
+
+  // In development mode, Next.js starts the dev server using child_process.fork with stdio set to 'inherit'.
+  // In order to capture the output, we need to ensure that the streams are pipeable and thus we perform a one-time
+  // monkey-patch of the ChildProcess.prototype.spawn method to override stdio[1] and stdio[2] to 'pipe'.
+  #ensurePipeableStreamsInFork () {
+    const originalSpawn = ChildProcess.prototype.spawn
+    ChildProcess.prototype.spawn = function (options) {
```

Comment:
Can we detect what is actually being spawned?

Reply:
Done, what about now?